.travis.yml

```diff
@@ -5,6 +5,7 @@ language: ruby
 cache: bundler
 before_install:
   - rm -f Gemfile.lock
+  - gem install bundler -v '< 2'
```

Comment:
I don't think that this line is required?

Reply:
It might not be anymore - there was an issue with the acceptance tests back in January which prompted this change. It looks like they still don't complete without timeouts on CentOS. I'll remove it and see what happens

This particular update occurred because I force pushed the branch after rebasing to the latest changes last week so there's nothing new here yet. I split out the correction to issue #459 into pull #470 as a stepping stone to making this pull less complicated.